packages/core/src/utils-hoist/worldwide.ts

```diff
@@ -12,41 +12,10 @@
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
-import type { Client, MetricsAggregator, Scope } from '../types-hoist';
-
+import type { Carrier, SentryCarrier, VersionedCarrier } from '../carrier';
 import type { SdkSource } from './env';
-import type { logger } from './logger';
 import { SDK_VERSION } from './version';
 
-interface SentryCarrier {
```

Comment:
this type was basically duplicated here - this comes from the days of core vs utils separation, I believe 😬